internal/dataplane/testdata/golden/fallback-config-kong-custom-entities/in.yaml

```diff
@@ -0,0 +1,277 @@
+# In this test case we have 4 independent sets of resources (each including Service, Ingress, Plugin, CustomEntity).
+# In each set, we make one of the objects broken by adding an annotation `test.konghq.com/broken: "true"`.
+
+# Set n. 1: Service s1, Ingress i1, Plugin p1, CustomEntity ce1. ce1 is broken.
+# We expect that the custom entity ce1 is not created, but the other resources are created.
+---
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    app: s1
+  name: s1
+  namespace: default
+  uid: ddc15c38-7f1b-41cb-8b57-46465a7f4427
+spec:
+  ports:
+    - protocol: TCP
+      port: 80
+      targetPort: 8080
+  selector:
+    app: s1
+---
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: i1
+  namespace: default
+  uid: 39ec2b6a-f919-4308-ad2f-07511ab424dd
+  annotations:
+    konghq.com/strip-path: "true"
+    konghq.com/plugins: "p1"
+spec:
+  ingressClassName: kong
+  rules:
+    -  http:
+        paths:
+          - path: /
+            pathType: Prefix
+            backend:
+              service:
+                name: s1
+                port:
+                  number: 80
+---
+apiVersion: configuration.konghq.com/v1
+kind: KongPlugin
+metadata:
+  namespace: default
+  name: p1
+  uid: e97ca177-6bd2-425f-b3b1-10a521ac02a0
+plugin: p1
```

Comment:
Looks like we cannot use fake (non-exist) plugins and custom entities when we run golden tests agianst Kong gateway.

Reply:
Yes, sorry I missed that. I've changed the tests to use `degraphql` plugin and `degraphql_routes` as custom entities. Also added `-ee` prefix to the golden tests directory name so it's properly treated as an enterprise test case. 

In https://github.com/Kong/kubernetes-ingress-controller/pull/6293 I'm resolving the issue that we had created to enable EE kongintegration tests.